code/modules/mob/living/simple_animal/abnormality/_abnormality.dm

```diff
@@ -93,6 +93,14 @@
 		addtimer(CALLBACK (datum_reference, .datum/abnormality/proc/RespawnAbno), 30 SECONDS)
 	..()
 
+/mob/living/simple_animal/hostile/abnormality/add_to_mob_list()
```

Comment:
It doesn't have to be its own proc.
Just add them to list on Initialize() and remove on Destroy()

Reply:
This is how literally every mob list is added to and is *built in*. I'm not making a new one here I'm using the existing one that adds them to the global mob list and expanding it for abnormalities.

Reply:
Also I'm pretty sure these are called at those exact times.

Reply:
Yeah it is. This is literally the best way to do this.